docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Pulp 2.11 Release Notes
+=======================
+
+Pulp 2.11.0
+===========
+
+New Features
+------------
+
+* For RPM content, a full sync will be forced if the sync configuration has been changed or content
```

Comment:
I believe that was part of the rationale for doing this (although it isn't really explicitly stated on the redmine issue).  I won't be able to get to that since my internship ends tomorrow however.